.eleventy.js

```diff
@@ -89,6 +89,12 @@ module.exports = function(eleventyConfig) {
         return `<span class="ff-tooltip" data-tooltip="${text}">${content}</span><span></span>`
     });
 
+    eleventyConfig.addPairedShortcode("blueCard", function(content) {
+        const md = new markdownIt();
+        let markdownContent = md.render(content);
```

Comment:
```suggestion
        const markdownContent = markdownIt.render(content);
```

Reply:
@Yndira-E Can you test this?

Reply:
I tried this approach before, but kept getting this error on the terminal
```
[11ty] Problem writing Eleventy templates: (more in DEBUG output)
[11ty] 1. Having trouble rendering liquid template ./src/whitepaper/accelerating-innovation-in-manufacturing-with-flowfuse.md (via TemplateContentRenderError)
[11ty] 2. markdownIt.render is not a function
```
Apparently this won't work because `render` is an instance method of markdownIt, not a static method. This means we need to create an instance of markdownIt before we can use the render method.

Here are the docs: https://markdown-it.github.io/markdown-it/